web-app/js/portal/filter/BooleanFilterPanel.js

```diff
@@ -41,8 +41,7 @@ Portal.filter.BooleanFilterPanel = Ext.extend(Portal.filter.BaseFilterPanel, {
     },
 
     _formatBoxLabel: function() {
-        var label = this.filter.label.split('_').join(' ').toTitleCase();
-        return label;
+        return this.filter.getDisplayLabel();
```

Comment:
It's a good idea to have a class separate to `Filter` for constructing CQL.  Anyway, I think we both agree it definitely doesn't belong in the UI.